skimage/morphology/footprints.py

```diff
@@ -74,6 +75,7 @@ def _odd_size(size, require_odd_size):
     return tuple(shape)
 
 
+@dispatchable
```

Comment:
The functions in this file are ones where I am not sure if they should be dispatched or not. In part because I don't know scikit-image well enough.

One tricky thing about these functions is that it is unclear what the type of the output should be. Imagine a cupy backend that implements these functions and outputs a cupy array. If it doesn't also implement every single consumer of these footprints, some of them will fail when given a cupy array. So maybe the backend should return a numpy array instead, but in that case there isn't much point in dispatching it.

Which makes me think maybe we should not dispatch these and backends should be clever enough to be able to take numpy arrays as input in functions like `skimage.morphology.opening`.

WDYT?

Reply:
From a "front-end" perspective, I could just say let the backend figure out if they want to do something with this or not. So it shouldn't hurt to just add the decorator here for now?